.postcssrc.json

```diff
@@ -1,7 +1,7 @@
 {
 	"plugins": {
 		"postcss-modules": {
-			"generateScopedName": "[local]__[contenthash]",
+			"generateScopedName": "parleychat-[local]__[contenthash]",
```

Comment:
Mag ik zo vervelend zijn dat ik dit parley-messaging of parley-messenger wil noemen...

Reply:
Done, e33fbdbc3914cf99687853611bebffe1c692c5d8

Reply:
Oops daar zat een typo in, nu echt gefixed 73dd742b23b34ceccb382213b3a0b68212467cfa